### PR TITLE
command: add membership counts to /list-private

### PR DESCRIPTION
**Pull request by zkat**

### CODE REVIEW

File: lib/commands/list-private.js [hunk 1]
```diff
@@ -17,7 +17,8 @@ function formatChannel (chan) {
 
 function displayGroupList (filter, groups) {
   const str = groups.reduce((acc, group) => (
-    `${acc}  ~ ${formatChannel(group)}\n`
+    // Subtracting 1 from `group.members.length` to exclude Wheelie.
+    `${acc}  ~ ${formatChannel(group)} (${group.members.length - 1})\n`
```
Comment (aredridel) [inline lib/commands/list-private.js:21]: Legit. I'd probably have filtered out the entry and then counted, but c'est l'code.

**Comment by aredridel** [comment]
LGTM

File: lib/commands/list-private.js (hunk 1 above)
Comment (zkat) [reply lib/commands/list-private.js:21]: The filter doesn't happen in this function. The `filter` argument here is purely to change the display message.

Comment (aredridel) [reply lib/commands/list-private.js:21]: Mmm. I mean filter out wheelie by name, then just count all.

Comment (zkat) [reply lib/commands/list-private.js:21]: oh. Yeah idk. This list can't be displayed at all without wheelie there, so :)

I'd need to track (and/or request) bot names across different slacks that might have different names. And then deal with potential weirdness in the match.

Comment (aredridel) [reply lib/commands/list-private.js:21]: Yeah. So LGTM :)

**Comment by zkat** [comment]
Thank you!

**Pull request merged**